README.md

```diff
@@ -834,7 +838,7 @@ Deploys a `BurnMintTokenPool` contract for a specific token and grants it the ne
 **Usage:**
 
 ```shell
-npx hardhat setup-burn-mint-pool --network <networkName> 
+npx hardhat setup-burn-mint-pool --network <networkName>
```

Comment:
for step 2 and 3 

1) should we clarify that BOTH pools are not required?
Devs are likely to just follow them in sequence...and should understand that they may want to make a choice - because there is a choice?

2) since the configure pool step will require deployment on a remote pool we should prepare them for that in this step.  For example if I am deploying only on source when i get to the configure pool step i will discover the --remote-pool (destination) address is needed...but i've only done on source chain!  So we should tell user here to deploy BOTH tokens _and_ pools on both networks by the time they get to step 4